### DISCUSSION TITLE
How are closed source tools harmful for civil society? Open source tools?

### DISCUSSION THREAD

**Discussion by rachan2023** [albertkun/21S-ASIAAM-191A#99] [Past Thinking Caps]
Although closed source tools are more secure and designed for user needs , closed source tools are harmful for civil society because they serve as a barrier to exclude people, such as marginalized communities, from using these tools and collaborating with one another. Oftentimes, closed source tools have monthly or annual fees that people have to pay in order to get access to these tools. For many people, especially those who are not economically stable, the cost may deter them from using these tools. In terms of digital mapping, since certain people can have access to these tools, it prevents everyone from collaborating with one another and creating different maps that tell their stories. On the other hand, open source tools are free and allow for collaboration and participatory mapping, but people can take advantage of these benefits and use them to harm people. On top of that, the code for open source tools are widely available, so people can copy all of the code and replicate it or find ways to hack these open source tools, which risk the safety of users.

**Comment by ghost** [comment]
Your response made me think of how ironic the research I participate in is. I produce maps and filter data for research regarding the Geography of Opportunity. I use ArcPro to produce my maps and conduct analysis which is a closed source tool. I am privileged to have access to this software but the communities, families, children, and demographics I am studying as a part of my civil rights-based research likely do not have that same opportunity and access to closed source tools. It is important for me to recognize this and also to be aware that while I may be using a closed source platform to bring attention to social discrepancies that my work could be more effective and meaningful if completed using QGIS.

**Comment by KenNL42** [comment]
Cost definitely becomes a huge factor from switching closed source tools to open source. If there is a project that is small scaled, then I also agree that this does not worth people to pay for it. In terms of open source tools being exploited to harm people, I believe it depends on how the open source tool is available for use. If the tool/data is there for people to use, then generally users can detect a malicious program directly from the source and maybe tell others to avoid it. In the end, if people create a program to exploit, then it may not matter if it is open or closed source.

**Comment by lvalles10** [comment]
I definitely agree with the concerns you raise about cost and preventing collaboration when it comes to closed source tools. I think in a capitalist society where people's data and code is so frequently monetized by big corporations, closed sourced tools become a weapon of defense against the people's revolution for data and collaboration. As long as closed sourced tools are around, our data and code are a commodity. It's realizations like that- the commodification of data and gatekeeping enabled by closed sourced tools- that are quite jarring like you mention above. 

I also agree with the concerns you raise about open source tools, but would also state how in terms of empowerment and liberation the accessible nature of open source tools seems to outweigh the "costs" of their misuse.
